packages/checkout/widgets-lib/src/lib/hooks/useInjectedProviders.ts

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Checkout, WalletProviderRdns } from '@imtbl/checkout-sdk';
+import { Web3Provider } from '@ethersproject/providers';
+import {
+  EIP1193Provider,
+  EIP6963ProviderDetail,
+  getPassportProviderDetail,
+  InjectedProvidersManager,
+} from '../provider';
+
+export interface UseInjectedProvidersParams {
+  checkout: Checkout | null;
+}
+
+let passportWeb3Provider: Web3Provider;
+const processProviders = (checkout: Checkout | null, injectedProviders: EIP6963ProviderDetail<EIP1193Provider>[]) => {
+  // Apply wallet providers allowlist
+  const allowedWalletRdns = [
```

Comment:
Can this Rdns be set from config? give that some are disable now, I'll think this list changes overtime?

Reply:
Yep good call, that was the plan. Just not in this PR.